app/experimenter/settings_test.py

```diff
@@ -11,3 +11,7 @@
 }
 
 HOSTNAME = "experimenter.moz"
+
+EMAIL_REVIEW = "testreview@example.com"
+EMAIL_SHIP = "testship@example.com"
+EMAIL_SENDER = "sender@example.com"
```

Comment:
These are set now so that when *you* run the unit tests they get the same, constant, email addresses and does *not* depend on personal `.env` files.